src/Bicep.LangServer/Completions/BicepCompletionContext.cs

```diff
@@ -25,6 +26,22 @@ public record FunctionArgumentContext(
             FunctionCallSyntaxBase Function,
             int ArgumentIndex
         );
+        private static readonly CompositeSyntaxPattern ExpectingImportSpecification = CompositeSyntaxPattern.Create(
+            "import |",
+            "import |'kuber'",
+            "import 'kuber|'");
+
+        private static readonly CompositeSyntaxPattern ExpectingImportWithOrAsKeyword = CompositeSyntaxPattern.Create(
+            "import 'kubernetes@v1' |",
```

Comment:
Would be good to have the version portion of the syntax match the new semver format.